src/Workspaces/SharedUtilitiesAndExtensions/Workspace/CSharp/LanguageServices/CSharpSyntaxGeneratorInternal.cs

```diff
@@ -118,14 +120,14 @@ internal static SyntaxTokenList GetParameterModifiers(RefKind refKind, bool forF
         => refKind switch
         {
             RefKind.None => [],
-            RefKind.Out => [SyntaxFactory.Token(SyntaxKind.OutKeyword)],
-            RefKind.Ref => [SyntaxFactory.Token(SyntaxKind.RefKeyword)],
+            RefKind.Out => [OutKeyword],
+            RefKind.Ref => [RefKeyword],
             // Note: RefKind.RefReadonly == RefKind.In. Function Pointers must use the correct
             // ref kind syntax when generating for the return parameter vs other parameters.
             // The return parameter must use ref readonly, like regular methods.
-            RefKind.In when !forFunctionPointerReturnParameter => [SyntaxFactory.Token(SyntaxKind.InKeyword)],
-            RefKind.RefReadOnly when forFunctionPointerReturnParameter => [SyntaxFactory.Token(SyntaxKind.RefKeyword), SyntaxFactory.Token(SyntaxKind.ReadOnlyKeyword)],
-            RefKind.RefReadOnlyParameter => [SyntaxFactory.Token(SyntaxKind.RefKeyword), SyntaxFactory.Token(SyntaxKind.ReadOnlyKeyword)],
+            RefKind.In when !forFunctionPointerReturnParameter => [InKeyword],
+            RefKind.RefReadOnly when forFunctionPointerReturnParameter => [RefKeyword, ReadOnlyKeyword],
+            RefKind.RefReadOnlyParameter => [RefKeyword, ReadOnlyKeyword],
```

Comment:
it's really nice how much some code gets cleaner thans to features like collectoin expressions and these simple static tokens.